appveyor.yml

```diff
@@ -18,7 +18,7 @@ environment:
   # Disable the .NET first time experience to skip caching NuGet packages and speed up the build.
   DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
   # Use latest version of current used .NET version.  6.0 could use the latest of 6.0.  See .\Scripts\Update-DotnetVersion.ps1
-  GE_USE_LATEST_DOTNET: false
+  GE_USE_LATEST_DOTNET: true
```

Comment:
I guess this doesn't match the PR description.

Reply:
Yes it does. It eliminates us managing and allows the build server to build with 6.0/latest.

Reply:
Reading it again, yes, the description is not incorrect.
But that it always builds the with the latest, should be clearly stated in the issue description.

Reply:
Yep. I don't like the formatting but if people do use script it will prevent a big dif. I would love to adjust script to make it pretty but all the ways to do that looked hacky and I didn't like the ways. So I chose to just let script be authority of formatting.

Reply:
![Screenshot_20221115-191316_GitHub.jpg](https://user-images.githubusercontent.com/3629489/202051983-1fad14c4-8270-4cba-8ac0-d8bb4475e8f1.jpg)